pacta/build_with_tag.sh

```diff
@@ -69,7 +79,6 @@ do
     rm -rf "$repo"
 done
 
-parent="$(dirname $(which $0))"
 image_tar_gz="${parent}/2dii_pacta.tar.gz"
```

Comment:
think you changed this syntax `${parent}`, to this syntax `$parent` everywhere else?

Reply:
Yeah, here I was afraid of the "/". Should it be fine?

Reply:
hmmmmm... good point... not sure

I guess this is also one of those places you should quote, quote, quote just to be safe, i.e. `""${parent}"/2dii_pacta.tar.gz"`

but don't take my word for it, experimenting/testing is the only way to know for sure I guess